tests/migrations/test_runner.py

```diff
@@ -190,21 +190,21 @@ def get_sampling_key() -> str:
     )
 
     assert get_sampling_key() == ""
-    generate_transactions(5)
```

Comment:
Why this change ? This is meant to test a data migration happening in `0002_transactions_onpremise_fix_orderby_and_partitionby`

Reply:
Oh I see. OK well the test was failing because the method and referer fields were missing. I figured the correct way to fix that was to run the migrations, but I'll change it back.

Reply:
OK I changed this in a way that I think makes sense.